utilities/install_requirements.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+import sys
+
+
+in_venv = getattr(sys, "real_prefix", getattr(sys, "base_prefix", sys.prefix)) != sys.prefix
+pip_call = [sys.executable, "-m", "pip"]
+pip_install = pip_call + ["install"]
+
+if not in_venv:
+    pip_install.append("--user")
+
+subprocess.check_call([*pip_install, "pip", "-U"])
+subprocess.check_call([*pip_call, "uninstall", "depthai", "--yes"])
+subprocess.check_call([*pip_install, "-r", "requirements.txt"])
+try:
+    subprocess.check_call([*pip_install, "-r", "requirements-optional.txt"])
+    if sys.platform == "linux":
+        print(f'$ sudo apt install python3-tk')
+except subprocess.CalledProcessError as ex:
+    print(f"Optional dependencies were not installed (exit code {ex.returncode})")
```

Comment:
I'd install python3-tk in this case...
Have you tested this. This seems like it just uninstalls the depthai as of right now...
Also requirements-optional.txt aren't available...

Anyway, in this case I think best thing would be moving the `examples/install_requirements.py` down to `/install_requirements.py` and adding an argument IFF only specific requirements are needed (eg docs/utilities/examples, although that can be added later). Otherwise same principle applies - installs depthai, dependencies, etc...

CC: @Erol444 @szabi-luxonis on above